src/magic_spell_effect.cpp

```diff
@@ -722,3 +723,15 @@ void spell_effect::flashbang( const spell &sp, Creature &caster, const tripoint
     explosion_handler::flashbang( target, caster.is_avatar() &&
                                   !sp.is_valid_target( valid_target::target_self ) );
 }
+
+void spell_effect::map( const spell &sp, Creature &caster, const tripoint & )
+{
+    const avatar *you = caster.as_avatar();
+    if( !you ) {
+        // revealing the map only makes sense for the avatar
+        return;
+    }
+    const tripoint center = you->global_omt_location();
+    const bool new_map = overmap_buffer.reveal(
```

Comment:
New_map is unused?
src/magic_spell_effect.cpp:735:16: error: unused variable ‘new_map’ [-Werror=unused-variable]
Are you missing a return value?